recipes/oboe/all/conanfile.py

```diff
@@ -40,13 +44,16 @@ def validate(self):
         if self.settings.os != "Android":
             raise ConanInvalidConfiguration("oboe supports Android only")
         if self.settings.compiler.get_safe("cppstd"):
-            check_min_cppstd(self, 17)
+            check_min_cppstd(self, self._min_cppstd)
 
     def source(self):
         get(self, **self.conan_data["sources"][self.version], strip_root=True)
 
     def generate(self):
         tc = CMakeToolchain(self)
+        if not valid_min_cppstd(self, self._min_cppstd):
+            tc.variables["CMAKE_CXX_STANDARD"] = self._min_cppstd
+        tc.cache_variables["CMAKE_POLICY_DEFAULT_CMP0077"] = "NEW"
```

Comment:
Why do you need this? I don't see any reason looking at upstream CMakeLists

Reply:
Just to be safe, due to `cmake_minimum_required(VERSION 3.4.1)`. I don't like checking whether the values are actually overwritten as "CACHE" or not in CMakeLists.txt, as it's prone to change at any time.